README.md

```diff
@@ -135,6 +135,53 @@ Parcelgen supports writing and reading the following types to a Parcel:
 }
 ```
 
+### Using Enums
+
+
+Parcelgen provides full support for working with enums. You can add enum fields to your models following the next steps:
+
+**1. Changes required to your .json file**
+
+* create the enum class in any .java file you want and import it
```

Comment:
Can you please make sure that you capitalize your sentences - even the ones right after the bullet point should be capitalized. After that I'll merge - thanks for writing this up!